Mathlib/Algebra/Group/Hom/Instances.lean

```diff
@@ -357,9 +361,8 @@ variable {R S : Type*} [NonUnitalNonAssocCommSemiring R]
 
 namespace AddMonoid.End
 
-lemma comm_mulRight_eq_mulLeft : mulRight = (mulLeft : R →+ AddMonoid.End R) := by
-  ext a
-  exact mulRight_eq_mulLeft_of_commute _ (Commute.all _)
+lemma mulLeft_eq_mulRight : mulLeft = (mulRight : R →+ AddMonoid.End R) :=
```

Comment:
I put `mulLeft` on the right because I considered that the simpler form. That may be a matter of taste though?

Reply:
Yes please. Thanks.

Reply:
Done